### ISSUE TITLE
bicycle=use_sidepath is too strong

### ISSUE THREAD

**Issue by c-h-r-i-s-t-i-a-n** [osmandapp/OsmAnd#19915]
### Describe the routing engine used (required)

- [x] OsmAnd's in-app offline routing
- [ ] Any online routing provider (YOURS, openrouteservice, OSRM, etc.)

### Describe your routing Profile (required)

Osmand bicycle

### Describe your start and end points (required)

correct routing in OSRM and Graphhopper: 

https://www.openstreetmap.org/directions?engine=graphhopper_bicycle&route=51.94078%2C8.26129%3B51.93967%2C8.26041#map=19/51.94023/8.26084

### Describe the actual route (required)

Osmand refused to route the cyclist 10 meters via the main road, instead goes over the greens:

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/57685590/f1f3f743-8581-4d84-82a7-e48d6507a2ac"  width="300" height="auto"/> </th>
  </tr>
</table>

### Describe the expected route (required)

OSMAND should add only a SMALL penalty to roads having bicycle=use_sidepath. 

correct routing in OSRM and Graphhopper: 

https://www.openstreetmap.org/directions?engine=graphhopper_bicycle&route=51.94078%2C8.26129%3B51.93967%2C8.26041#map=19/51.94023/8.26084

### Describe what Maps you used (online or offline) (required)

- [x] Offline maps offered within the OsmAnd app for download. 
- [ ] Online (tile / raster) maps 

### Your Environment (required)

```
OsmAnd Version: osmand+ from play store (newest)
Android/iOS version: 11
Device model: Redmi note 13
```

**Comment by zipav** [comment]
OsmAnd Nightly 4.8.0#2380m, released: 2024-05-24

Reproduced. 

<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/123936521/32e8d4e4-9571-43a1-9c3f-a91b89b0f425"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by DmitryAlexei** [comment]
fixed on web https://test.osmand.net/map/navigate/?start=51.940822,8.261300&end=51.939688,8.260396&profile=bicycle#18/51.94059/8.25968
+ in the OsmAnd~ 4.8.0#2583m, released: 2024-06-25 and OsmAnd TestFlight 4.7.7.5
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/ae0c6e59-aeb0-4bda-b89c-50d89a0bee84"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/116079541/d68833a5-7d85-4afa-a2e6-2f896d837015"  width="300" height="auto"/> </td>
  </tr>
</table>
